lib/p2p/Peer.ts

```diff
@@ -216,6 +216,9 @@ class Peer extends EventEmitter {
     if (clientType === SwapClientType.Raiden) {
       return this.nodeState.raidenAddress;
     }
+    if (clientType === SwapClientType.Connext) {
```

Comment:
Might be nice to combine this and the if statements above into a switch statement, now that there are more than 2 possibilities.